hackerank/js-practice/angry-professor.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+// Complete the angryProfessor function below.
+function angryProfessor(studentsThreshold, arrivalTime) {
+  const arrivedStudents = arrivalTime.filter((studentTime) => studentTime <= 0);
+  if (arrivedStudents.length < studentsThreshold) {
+    return "YES";
+  }
+  return "NO";
+}
+
+function main(studentsThreshold, arrivalTime, response) {
+
+  let result = angryProfessor(studentsThreshold, arrivalTime);
+  if (result === response) {
+    console.log("ok");
+  } else if (result !== response) console.log("not ok");
```

Comment:
You don't need to add the conditional `if (result !== response)`. 

The else explicit that the `result` is not equal to `response`.